api-report/fluid-static.api.md

```diff
@@ -46,6 +46,7 @@ export class FluidContainer extends TypedEventEmitter<IFluidContainerEvents> imp
     dispose(): void;
     get disposed(): boolean;
     get initialObjects(): Record<string, IFluidLoadable>;
+    get isDirty(): boolean;
```

Comment:
Could we consider a different name for this API, like `hasPendingOps` or something like that? "Dirty" isn't as descriptive.